testsuite/cli/src/client_proxy.rs

```diff
@@ -756,7 +752,7 @@ impl ClientProxy {
 
```

Comment:
can we locally test the transfer command on CLI to make sure it still works? https://github.com/libra/libra/blob/master/testsuite/cli/src/transfer_commands.rs#L25-L38 and update potentially outdated CLI outputs 😄

Reply:
Checked it out and  it still works! For `transfer_commands.rs`, the min/max # of parameters hasn't changed and the `get_params_help` actually didn't mention `auth_key_prefix`, so I think we're good.

Reply:
thanks for checking! lgtm from client side, just gotta fix the failing tests